cpp/demo/hyperelasticity/main.cpp

```diff
@@ -164,8 +164,10 @@ int main(int argc, char* argv[])
   // .. code-block:: cpp
 
   // Create mesh and define function space
-  std::array<geometry::Point, 2> pt
-      = {geometry::Point(0.0, 0.0, 0.0), geometry::Point(1.0, 1.0, 1.0)};
+  std::array<Eigen::Vector3d, 2> pt;
```

Comment:
Does `std::array<Eigen::Vector3d, 2> pt{(0, 0, 0), (1, 1, 1)};` work?

Reply:
Unfortunately not. Very little works for Eigen initialisation.

Reply:
`std::array<Eigen::Vector3d, 2> pt{Eigen::Vector3d(0.0, 0.0, 0.0),
                                    Eigen::Vector3d(1.0, 1.0, 0.0)};`. How about that?